Automattic/jetpack
# Slideshow block: remove dependency on Lodash

The slideshow block frontend script currently depends on Lodash for some trivial collection manipulation. This PR replaces that usage with vanilla ES2015+, removing the need for the dependency.

This PR should remove ~70KB (25KB over the wire) of JS on every page with a slideshow block.

Note to @anomiex: this PR removes the need for one of the entries in the #28626 allowlist

## Proposed changes:
* Remove dependency on Lodash from slideshow block frontend code

### Other information:

- N/A Have you written new tests for your changes, if applicable?

## Jetpack product discussion
None, this is a simple performance improvement.

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
- Create a page with a slideshow block
- Ensure that lodash isn't loaded when you load the page
- Smoke-test the slideshow block and ensure that everything works correctly
